src/diffbot.js

```diff
@@ -308,14 +311,36 @@ class Diffbot {
         if (options.maxToProcess != undefined)
           diffbot_url += `&maxToProcess=${options.maxToProcess}`;
 
+        if (options.notifyEmail)
+          diffbot_url += `&notifyEmail=${options.notifyEmail}`;
+
         if (options.notifyWebhook)
           diffbot_url += `&notifyWebhook=${options.notifyWebhook}`;
 
         // TODO: add supprt for the other optional params
         // urlCrawlPattern, urlCrawlRegEx, urlProcessPattern, urlProcessRegEx, pageProcessPattern
         // and possibly some of the others (https://docs.diffbot.com/docs/en/api-crawlbot-api)
 
-        return fetch(diffbot_url, 'POST');
+        return request(diffbot_url, 'POST');
+      },
+      //To check status of existing job
+      status: async function(options){
+        console.log(options.jobname);
+        if (!options.jobname) {
+          throw new Error('missing name');
+        }
+        let diffbot_url = `https://api.diffbot.com/v3/crawl?token=${this.token}`
+          + `&name=${encodeURIComponent(options.jobname)}`;
+
+        if (options.apiUrl) {
+          diffbot_url += `&apiUrl=${encodeURIComponent(options.apiUrl)}`;
+        } else {
+          diffbot_url += `&apiUrl=${encodeURIComponent('https://api.diffbot.com/v3/analyze?mode=auto')}`;
+        }
```

Comment:
I don't see this param in the docs? www.diffbot.com/dev/docs/crawl/api.jsp#details